pkgs/development/python-modules/duckdb-engine/default.nix

```diff
@@ -35,18 +36,22 @@ buildPythonPackage rec {
   ];
 
   preCheck = ''
-    export HOME="$(mktemp -d)"
+    HOME="$(mktemp -d)"
   '';
 
-  # this test tries to download the httpfs extension
   disabledTests = [
+    # this test tries to download the httpfs extension
     "test_preload_extension"
+    # test should be skipped based on sqlalchemy version but isn't and fails
+    "test_commit"
   ];
 
   nativeCheckInputs = [
     pytestCheckHook
     hypothesis
     ipython-sql
+    # TODO(cpcloud): include pandas here when it supports sqlalchemy 2.0
```

Comment:
https://github.com/NixOS/nixpkgs/pull/225662 is waiting for the next python-updates cycle.